packages/react/src/components/VideoPlayer/__stories__/VideoPlayer.stories.js

```diff
@@ -38,7 +38,7 @@ storiesOf('Components|VideoPlayer', module)
             <VideoPlayer
               inverse={inverse}
               videoId="0_uka1msg4"
-              showDescription={boolean('Show description', true)}
+              showCaption={boolean('Show caption', true)}
```

Comment:
Adding prop name in the description:

```suggestion
              showCaption={boolean('Show caption (showCaption)', true)}
```